lib_nbgl/src/nbgl_use_case.c

```diff
@@ -893,13 +1117,60 @@ uint8_t nbgl_useCaseGetNbPagesForTagValueList(const nbgl_layoutTagValueList_t *t
         // upper margin
         nbPairsInPage = nbgl_useCaseGetNbTagValuesInPage(nbPairs, tagValueList, i, &tooLongToFit);
         i += nbPairsInPage;
-        setNbPairs(nbPages, nbPairsInPage, tooLongToFit);
         nbPairs -= nbPairsInPage;
         nbPages++;
     }
     return nbPages;
 }
 
+uint8_t nbgl_useCaseGetNbPagesForContent(const nbgl_content_t *content, uint8_t pageIdxStart)
+{
+    uint8_t nbElements = 0;
+    uint8_t nbPages    = 0;
+    uint8_t nbElementsInPage;
+    uint8_t elemIdx = 0;
+    bool    flag;
+
+    nbElements = getContentNbElement(content);
+
+    while (nbElements > 0) {
+        if (content->type == TAG_VALUE_LIST) {
+            nbElementsInPage = nbgl_useCaseGetNbTagValuesInPage(
+                nbElements, &content->content.tagValueList, elemIdx, &flag);
+        }
+        else {
+            nbElementsInPage = MIN(
+                3, nbElements);  // TODO hardcoded to 3 for now but should be dynamically computed
```

Comment:
Better to have a real function/define easier to check and update later...

Reply:
@nroggeman-ledger gave me a better mapping. It will be changed to a dynamic version at some point!